generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java

```diff
@@ -79,11 +79,11 @@ public void run(final Raml raml, final GeneratorConfig configuration) {
     therefore we need all messages with consecutive numbers there. Messages need to be in correct order in order to update the view correctly.
     */
 
-    private boolean shouldGenerateEventFilter(final Resource resource, final MessagingAdapterBaseUri baseUri) {
+    static boolean shouldGenerateEventFilter(final Resource resource, final MessagingAdapterBaseUri baseUri) {
         return EVENT_LISTENER.equals(baseUri.component()) && !containsGeneralJsonMimeType(resource.getActions());
     }
 
-    private boolean shouldListenToAllMessages(final Resource resource, final MessagingAdapterBaseUri baseUri) {
+    static boolean shouldListenToAllMessages(final Resource resource, final MessagingAdapterBaseUri baseUri) {
```

Comment:
As above

Reply:
Moved to a separate class JmsEndPointGeneratorUtil